docs: Add a KLC in a nuthell section to the beginning of the doc (#127)
keptn/lifecycle-toolkit: README.md

This approach is also known as a _Level 3 Operator_
It targets cloud native applications deployed in Kubernetes and managed via modern approaches like GitOps.
It is not yet another deployment tool but an add-on that can easily integrate into your application management and CD ecosystem
to enable cloud native app operations.

Keptn Lifecycle Controller:

- Provides a standardized way to ensure that applications are running perfectly:
  Pre-Deployment, Post-Deployment, and Application Health Checks.
- Supports connecting Event Hooks to other stages of the application lifecycle.
- Allows running Pre- and Post-Deployment Tasks and Analysis without Pipelines.
  External pipeline engines can be connected for more complex automation and synchronous operations.
- Includes a Typescript based [functions runtime](./functions-runtime) for micro-scripting of event handlers.
  Scripts could shared 
- Later: allows handling of Kubernetes events, e.g. Deployments once implemented by the [Keptn App Lifecycle working group](https://github.com/keptn/wg-app-lifecycle) or CloudEvents

### References

> - TODO: Overview slidedeck

Author: ```suggestion
- [Overview slidedeck](https://docs.google.com/presentation/d/1satTEbjIoCZXLyx40i-IdV3hWd8ctnMz/edit?usp=sharing&ouid=104704987056219028407&rtpof=true&sd=true)
```

> The goal of this prototype is to introduce a more “cloud-native” approach for pre- and post-deployment, as well as the concept of application health checks.

Author: ```suggestion
The goal is to introduce a more “cloud-native” approach for pre- and post-deployment, as well as the concept of application health checks.
```

> It is an experimental project, under the umbrella of the [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).

Author: How about now?

```suggestion
It is developed under the umbrella of the [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
```

## Keptn Lifecycle Controller in a nutshell

The idea for **Keptn Lifecycle Controller** is to allow managing the App Lifecycle without writing operator code.
This approach is also known as a _Level 3 Operator_
It targets cloud native applications deployed in Kubernetes and managed via modern approaches like GitOps.
It is not yet another deployment tool but an add-on that can easily integrate into your application management and CD ecosystem
to enable cloud native app operations.

Keptn Lifecycle Controller:

- Provides a standardized way to ensure that applications are running perfectly:
  Pre-Deployment, Post-Deployment, and Application Health Checks.

> - Supports connecting Event Hooks to other stages of the application lifecycle.

Reviewer: This is a bit unclear to me ... What are Event Hooks in KLC?

Author: It comes from older slides.
How about: "Connects to other tools via CRDs or webhooks"?
